components/benefit_markets/app/models/benefit_markets/products/product.rb

```diff
@@ -371,6 +371,14 @@ def create_copy_for_embedding
         end
       end
 
+      def covers_pediatric_dental?
+        benefits = qhp.qhp_benefits
+
+        benefits.where(benefit_type_code: 'Dental Check-Up for Children').first&.is_benefit_covered == 'Covered' &&
```

Comment:
Is there an enum we could use instead of `Covered`?

Reply:
hmm, there isn't. But, I can add one i guess. I will add it if i have to make another commit.